src/constants/routes.constants.coffee

```diff
@@ -5,7 +5,7 @@ routes_Names =
   views:
     # defined via dedicated state: '/user/article'
     guest     : ['about',  'features', 'home', 'login', 'pwd_forgot', 'sign_up']
-    user_Root : ['docs','terms-and-conditions']
-    user_User : ['main','index', 'articles']
+    user_Root : ['docs' , 'terms_and_conditions']
```

Comment:
the route route name needs to be terms-and-conditions with dashes '-' and not with underscore because that is how the  file is called in TM_Flare.

![screen shot 2015-10-15 at 14 28 24](https://cloud.githubusercontent.com/assets/1732035/10526581/06600de8-7349-11e5-9748-5f18d55e8061.png)